internal/server/index.go

```diff
@@ -14,9 +15,40 @@ import (
 
 var alphaNumericRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
 
-// ServeIndex serve the index.html file with nonce generated for supporting
-// restrictive CSP while using material-ui.
-func ServeIndex(publicDir string) fasthttp.RequestHandler {
+// ServeEmbeddedIndex serve the index.html file with nonce generated for supporting
+// restrictive CSP while using material-ui from the embedded virtual filesystem.
+func ServeEmbeddedIndex(publicDir string) fasthttp.RequestHandler {
+	f, err := br.Open(publicDir + "/index.html")
+	if err != nil {
+		logging.Logger().Fatalf("Unable to open index.html: %v", err)
+	}
+
+	b, err := ioutil.ReadAll(f)
+	if err != nil {
+		logging.Logger().Fatalf("Unable to read index.html: %v", err)
+	}
+
+	tmpl, err := template.New("index").Parse(string(b))
+	if err != nil {
+		logging.Logger().Fatalf("Unable to parse index.html template: %v", err)
```

Comment:
Write an error response and return as suggested above. Same thing in the other handler.